public/css/item/downtime.less

```diff
@@ -62,23 +69,12 @@
     height: ~"calc(100% + 1px)"; // +1px due to the border added exclusively for the first item
   }
 
-  .visual {
-    justify-content: center;
-    flex-shrink: 0;
-    line-height: 1em;
-    margin-right: .5em;
-    padding: .5em .25em;
-    text-align: center;
-    width: 6em;
-
-    strong {
-      font-size: 1.5em;
-      line-height: 1em;
-    }
+  &.in-effect .main {
```

Comment:
This should not depend on `.in-effect`. This is about the fact that there is a progress bar. In minimal mode there is none. But the visual still depends on `.in-effect`, so we should make this dependent on the existence of the progress bar. Use `&:has(.progress)` instead